tabo-hdinsight/Azuredeploy.json

```diff
@@ -0,0 +1,555 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+  },
+  "variables": {
+    "TaboVnetId": "[resourceId('Microsoft.Network/virtualNetworks', variables('TaboVirtualNetworkName'))]",
+    "TaboSubnetRef": "[concat(variables('TaboVnetIdnetId'), '/subnets/', variables('TaboSubnetName'))]",
+    "TaboLocation": "[resourceGroup().location]",
+    "TaboVirtualMachineName": "[concat('v', uniquestring(resourcegroup().id))]",
+    "TaboVirtualmachinesize": "standard_A3",
+    "TaboAdminUsername": "azureuser",
+    "TaboAdminPassword": "Password@123",
+    "TaboVirtualNetworkName": "[concat('vn', uniquestring(resourcegroup().id))]",
+    "TaboStorageAccountName": "[concat('stor', uniquestring(resourcegroup().id))]",
+    "TaboStorageAccountType": "Standard_LRS",
+    "TaboAddressPrefix": "10.2.0.0/24",
+    "TaboSubnetName": "[concat('sub', uniquestring(resourcegroup().id))]",
+    "TaboSubnetPrefix": "10.2.0.0/24",
+    "TaboPublicIpAddressName": "[concat('pip', uniquestring(resourcegroup().id))]",
+    "TaboPublicIpAddressType": "static",
+    "TaboSqlConnectivityType": "Public",
+    "TaboSqlPortNumber": "1433",
+    "TaboSqlStorageDisksCount": 1,
+    "TaboSqlStorageWorkloadType": "General",
+    "TaboSqlStorageDisksConfigurationType": "RAID0",
+    "TaboSqlStorageStartingDeviceId": 1,
+    "TaboSqlAutopatchingDayOfWeek": "Sunday",
+    "TaboSqlAutopatchingStartHour": "02:00",
+    "TaboSqlAutopatchingWindowDuration": "60",
+    "TaboSqlAuthenticationLogin": "azureuser",
+    "TabooSqlAuthenticationPassword": "Password@123",
+    "TaboNetworksecuritygroupname": "[concat('nsg', uniquestring(resourcegroup().id))]",
+    "TaboNetworkinterfacename": "[concat('nic', uniquestring(resourcegroup().id))]",
+    "TaboVmdns": "[concat('tab', uniquestring(resourcegroup().id))]",
+    "TaboScriptfolder": ".",
+    "TaboScriptfilename": "sqlnew.ps1",
+    "TaboServiceBusNamespaceName": "[concat('sb', uniquestring(resourcegroup().id))]",
+    "TaboEhVersion": "2015-08-01",
+    "TaboDefaultSASKeyName": "RootManageSharedAccessKey",
+    "TaboDefaultAuthRuleResourceId": "[resourceId('Microsoft.ServiceBus/namespaces/authorizationRules', variables('TaboServiceBusNamespaceName'), variables('TaboDefaultSASKeyName'))]",
+    "TaboEventhubin": "IotInputxx",
+    "TaboEventhubout": "Iotoutputxx",
+    "TaboInKeyP": "DG4TcCs+XnDN83VZcT2e5NfVeSFvTuiYUiWtXhxnJ2s=",
```

Comment:
What format are Event Hub Keys? The CI can generate ssh-format public keys (GEN-SSH-PUB-KEY), but at the moment that's it; it's okay if the PR doesn't pass CI due to this; we can still merge it in as long as you've tested it manually :)